### ISSUE TITLE
Wrong AUTO_INCREMENTs when generating update migrations

### ISSUE THREAD

**Issue by zolex** [bizley/yii2-migration#4]
When a table has a (composite) primary key (in my case both WITHOUT AUTO_INCREMENT) the update and update-all command appends AUTO_INCREMENT to each property which is in the key (the create and create-all command don't do that). so if you have a table with a (composite) key it is generated correctly by the create command: ie. `$this->addPrimaryKey('PRIMARY', '{{%table}}', ['property_a,'property_b']);` but if you run the update command afterwards it somehow generates the following wrong statements which add the AUTO_INCREMENT to each property in the (composite) key:
`$this->alterColumn('{{%table}}', 'property_a', $this->...->append('AUTO_INCREMENT'));` and
`$this->alterColumn('{{%table}}', 'property_b', $this->...->append('AUTO_INCREMENT'));`

this also happens if a property was generated by the create command with `$this->...->append('PRIMARY KEY')`. then the update command generates a `$this->...->append('AUTO_INCREMENT')`

it seems the updater assumes that every key should be auto_increment, which is wrong.

PS: happening with mysql.

**Comment by bizley** [comment]
Could you paste here the SQL with create statement for this table so I can reproduce this bug exactly?

**Comment by zolex** [comment]
here is one create which results in this error

```
CREATE TABLE `auth_assignment` (
  `item_name` varchar(64) COLLATE utf8_unicode_ci NOT NULL,
  `user_id` int(11) unsigned NOT NULL,
  `created_at` varchar(20) COLLATE utf8_unicode_ci NOT NULL,
  `gueltig_bis_datetime` datetime DEFAULT NULL,
  PRIMARY KEY (`item_name`,`user_id`),
  KEY `fk_auth_login_user_id` (`user_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
```

**Comment by bizley** [comment]
Sorry, one more thing needed - the altered version of the table (SQL statement) - so I can run update on it.

**Comment by zolex** [comment]
there is no altered version. the create command generates the first proper migration with create table. and when you then run the update command (without changing the table) it adds AUTO_INCREMENTS to the key properties.

**Comment by bizley** [comment]
Confirmed bug, thank you for reporting.

**Comment by bizley** [comment]
Fixed in [2.1.2](https://github.com/bizley/yii2-migration/releases/tag/2.1.2).
